CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -462,13 +496,17 @@ namespace Kartaclysm
 
 	void ComponentKartController::ArmorPlate(int p_iArmorStack)
 	{
-		// TODO: I think maybe there should only actually be 4 stacks *shrug*
 		m_iMaxSpeedCoreStat = 1 + p_iArmorStack;
 		m_iAccelerationCoreStat = 5 - p_iArmorStack;
 		m_iHandlingCoreStat = 5 - p_iArmorStack;
```

Comment:
I'm not sure if the 5 represents max stacks on the ability or some max number for the stats. You changed the amount of max stats on the armor to be 4, not 5, but this hasn't changed. If it is the stacks, this value should change with the data in case we wanted to change the number of stacks again.